app/assets/javascripts/application.js

```diff
@@ -12,8 +12,14 @@
 //
 //= require jquery
 //= require jquery_ujs
-//= require foundation
+//= require foundation.core.js.es6
+//= require foundation.util.triggers.js
+//= require foundation.util.mediaQuery.js
+//= require foundation.responsiveMenu.js
+//= require foundation.responsiveToggle.js
 //= require vue
 //= require_tree .
 
-$(function(){ $(document).foundation(); });
+$(function(){ 
+  $(document).foundation();
+});
```

Comment:
Did we find out if this works with turbolinks? Normally instead of using document ready in rails you use 

```$( document ).on('turbolinks:load', function() {
  console.log("It works on each visit!")
})```